zcash_primitives/src/lib.rs

```diff
@@ -17,6 +17,7 @@ pub mod memo;
 pub mod merkle_tree;
 pub mod sapling;
 pub mod transaction;
+pub mod transparent;
```

Comment:
This module should still be behind the `transparent-inputs` feature flag, for those wallet vendors that do not choose to include transparent support, we don't want to require it.